opm/input/eclipse/Schedule/Schedule.hpp

```diff
@@ -45,6 +45,11 @@
 
 namespace Opm
 {
+    namespace Action {
+        class ActionX;
+        class PyAction;
+        class State;
+    };
```

Comment:
I didn't pay enough attention to this whilst reviewing, but the trailing semicolon on the closing brace of the namespace declaration is superfluous and will generate warnings in some elevated warning settings.